app/app/Http/Controllers/LocationManagementController.php

```diff
@@ -211,11 +226,103 @@ public function addNewLocationSave(Request $request)
 					);
 				}
 			}
+		
 		});
 
 		return view('pages.location_management.location_created', $view_data);
 	}
+public function editLocationSave(Request $request)
+	{
+		if(Input::has('location_id')){
+			$input_edit_id = Input::get('location_id');
+		}else{
+			$input_edit_id = "";
+		}
+	//	print_r(Input::get('location_id'));exit;
+		$user = BaseUser::getDbUser();
+		// perform some validation.
+		$validation_rules = array(
+			'name'           => 'required|between:2,255',
+			'longitude'           => 'numeric|required|between:-180,180',
+			'latitude'            => 'numeric|required|between:-90,90',
+			'phone_number'          => 'max:50',
+			'address'				=> 'max:255|required',
+			'external_web_url'		=> 'max:255|url',
+			'location_tags'			=> 'array',
+			'location_tags.*'		=> 'int|required' // Every array element is an integer.
+		);
+		$validator = Validator::make(Input::all(), $validation_rules);
+		$fields = ['name','address', 'phone_number',
+			'external_web_url', 'location_group_id'];
+		$view_data = $this->getLocationAddViewData($input_edit_id);
+		foreach ($fields as $fieldName) {
+			if ( Input::has($fieldName) ) {
+				$view_data['location']->{$fieldName} = Input::get($fieldName);
+			}
+			else {
+				$view_data['location']->{$fieldName} = '';
+			}
+		}
+		$location_group_id = Input::get('location_group_id');
+		if ( !is_numeric($location_group_id) ) {
+			$location_group_id = null; // convert things like '-' to null.
+		}
+		else {
+			$location_group_id = intval($location_group_id);
+		}
+		$location = $view_data['location'];
+		$location->location_group_id = $location_group_id;
+		//$view_data['locations'] = json_encode($this->getLocationsNear($location->longitude, $location->latitude));
```

Comment:
Remove this commented code.  There's no need to keep it around when it doesn't do anything useful.

Reply:
done